src/QuickMarcEditor/utils.test.js

```diff
@@ -577,115 +577,170 @@ describe('QuickMarcEditor utils', () => {
   });
 
   describe('formatMarcRecordByQuickMarcAction', () => {
-    it('should return original record if action is not "derive" or "create"', () => {
-      const record = {
-        records: [{
-          tag: '001',
-          content: 'some content',
-        }],
-      };
+    describe('when action is "edit"', () => {
+      it('should return original record if ', () => {
+        const record = {
+          records: [{
+            tag: '001',
+            content: 'some content',
+          }],
+        };
 
-      expect(utils.formatMarcRecordByQuickMarcAction(record, QUICK_MARC_ACTIONS.EDIT)).toEqual(record);
+        expect(utils.formatMarcRecordByQuickMarcAction(record, QUICK_MARC_ACTIONS.EDIT)).toEqual(record);
+      });
```

Comment:
In case of edit we pass the data unchanged, so we just need to make sure that function returns what it received. Added additional `updateInfo` to make sure that data is not lost too